deployments/warp_routes/stTIA/eclipse-stride-config.yaml

```diff
@@ -0,0 +1,30 @@
+# yaml-language-server: $schema=../schema.json
+options:
+  interchainFeeConstants:
+    - addressOrDenom: ibc/BF3B4F53F3694B66E13C23107C84B6485BD2B96296BB7EC680EA77BBA75B4801
```

Comment:
since this is the TIA address, should we set something else for stTIA?

Reply:
the fee is always in TIA - a bit weird but that's what stride wanted, and we can only really charge in one currency on a single IGP